src/System.Runtime.WindowsRuntime/src/System/Runtime/InteropServices/WindowsRuntime/MarshalingHelpers.cs

```diff
@@ -116,9 +119,22 @@ internal static IntPtr ConvertToNative(NotifyCollectionChangedEventArgs managedA
             try
             {
                 if (managedArgs.NewItems != null)
-                    newItemsIP = Marshal.GetComInterfaceForObject(managedArgs.NewItems, typeof(IBindableVector));
```

Comment:
`Marshal.GetComInterfaceForObject(managedArgs.NewItems, typeof(Windows.UI.Xaml.Interop.IBindableVector))` does not work?

Reply:
no, it doesn't. IBindableVector is a projected type, so you can't access it in managed code.

Reply:
Do you prefer me to change the visibility for IBindableVector from internal to public?

Reply:
```
Changing visibility would be my least preferred option. 

I guess having a local dummy type to just get the guid would work: 

```
    [ComImport]
    [Guid("xxx")]
    [WindowsRuntimeImport]
    internal interface IBindableVector_Stub
    {
    }
```

But I do not think it is strictly better than what you got here.

I think your change is fine (modulo the nit for IBindableVectorID name).

Reply:
update the guid name